twisted/internet/defer.py

```diff
@@ -48,7 +48,7 @@ class CancelledError(Exception):
 class TimeoutError(Exception):
     """
     This exception is deprecated.  It is used only by the deprecated
```

Comment:
Ah, you've encountered an interesting thing.  Part of the comment is wrong.  Just delete this sentence completely:
```
It is used only by the deprecated L{Deferred.setTimeout} method.
```
That comment is not true.  twisted.internet.defer.TimeoutError is used in other places, such as twisted/names/error.py